libs/langchain-openai/src/chat_models.ts

```diff
@@ -83,7 +81,7 @@ import {
 } from "./utils/openai-format-fndef.js";
 import { _convertToOpenAITool } from "./utils/tools.js";
 
-export type { AzureOpenAIInput, OpenAICallOptions, OpenAIChatInput };
```

Comment:
will removing this export mess some people up or better to have all azure exports from azure file directly?

Reply:
Yeah I think taking the opportunity to separate things would be nice